pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "gatorgrade"
+version = "0.1.0"
+description = "Python tool to execute GatorGrader"
+authors = ["Michael Abraham", "Jacob Allebach", "Liam Black", "Katherine Burgess", "Yanqiao Chen", "Ochirsaikhan Davaajambal", "Tuguldurnemekh Gantulga", "Anthony Grant-Cook", "Dylan Holland", "Peyton Kelly", "Luke Lacaria", "Lauren Nevill", "Jack Turner", "Daniel Ullrich", "Garrison Vanzin", "Rian Watson"]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4.0"
+typer = "^0.4.1"
+
+[tool.poetry.dev-dependencies]
+taskipy = "^1.10.1"
+pytest = "^7.1.1"
+pytest-cov = "^3.0.0"
+pylint = "^2.13.4"
+black = "^22.3.0"
+flake8 = "^4.0.1"
+mypy = "^0.942"
+pydocstyle = "^6.1.1"
+
+[tool.taskipy.tasks]
+test = { cmd = "pytest --cov-report term-missing --cov-fail-under=70  --cov-branch --cov=package tests/", help = "Run the pytest test suite" }
+lint = { cmd = "pylint package/ tests/" }
+flake8 = { cmd = "flake8 gatorgrade tests", help = "Run the flake8 checks for source code documentation" }
+mypy = { cmd = "poetry run mypy gatorgrade --show-error-codes", "run the mypy type checker for potential type errors" }
+black = { cmd= "black gatorgrade tests --check", help = "Run the black checks for source code format" }
+pydocstyle = { cmd = "pydocstyle gatorgrade tests", help = "Run the pydocstyle checks for source code documentation" }
+pylint = { cmd = "pylint gatorgrade tests", help = "Run the pylint checks for source code documentation" }
```

Comment:
Is this the same functionality as the `lint` task? The `lint` task is used in GitHub Actions (see `.github/workflows/main.yml` in PR #28), so if the `pylint` task duplicates the functionality, then the `lint` task should probably be kept instead of `pylint`.

Reply:
Yes that would make more sense, thank you.

Reply:
I think the goal is to include all linters in one task to make running them easier. This would mean that a pylint task is different from a lint task. Is there a specific preference on how to set this up?

Reply:
@Michionlion Yes, your approach is what I was referring to as well. I just wasn't sure how to describe it.  I've set it up like that for multiple projects and haven't had any problems on Mac or Linux. But I'm not sure if it could cause problems on windows either.